testdata/array_of_enums.go

```diff
@@ -4,6 +4,7 @@ const ArrayOfEnums = `{
     "$schema": "http://json-schema.org/draft-04/schema#",
     "properties": {
         "description": {
+            "properties": {},
```

Comment:
Now all these properties fields are popping up. I think when using maps it used to hide it before. Not sure we should proceed with this or no. Let me know. The tests are not passing yet, some are  still failing but I don't really want to throw time away if we do not agree to fix them.

The thing is that tests are broken on your master too, try `make test`:
`unknown import path "github.com/chrusty/protoc-gen-jsonschema/testdata": cannot find module providing package github.com/chrusty/protoc-gen-jsonschema/testdata`